### DISCUSSION TITLE
Remove option from fab from program code

### DISCUSSION THREAD

**Discussion by rdobrich** [nambicompany/expandable-fab#19] [Q&A] [answered]
Hi,
How can I remove (or hide) option from fab from program code?

**Comment by kabumere** [comment, marked as answer]
~~Hello @rdobrich,~~

~~Please see [this answer](https://github.com/nambicompany/expandable-fab/issues/12#issuecomment-748299013) to a previously asked question. If this is not quite what you're asking for, please elaborate your needs.~~

**UPDATE**:
To @rdobrich and anyone else interested in this feature: dynamically removable FabOptions has since been implemented in the library, starting from version `v1.1.1`. You can pull it (or later versions) from Maven Central, or clone the latest changes from the `master` branch on this repo. Feel free to start a discussion or create an issue if you have any questions, comments or bug reports!

**Reply by Maha97** [reply]
Where I can find the code for remove options dynamically ?
